Resources/public/js/views/ez-userprofileview.js

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-userprofileview', function (Y) {
+    'use strict';
+    /**
+     * Provides the User Profile View class
+     *
+     * @module ez-userprofileview
+     */
+    Y.namespace('eZ');
+
+    var CLASS_USER_PROFILE_HIDDEN = 'is-user-profile-hidden',
+        SELECTOR_USER_NAME = '.ez-user-profile-name',
+        SELECTOR_USER_ROLE = '.ez-user-profile-role',
+        SELECTOR_USER_IMAGE = '.ez-user-profile-image';
+
+    /**
+     * The User Profile View
+     *
+     * @namespace eZ
+     * @class UserProfileView
+     * @constructor
+     * @extends eZ.TemplateBasedView
+     */
+    Y.eZ.UserProfileView = Y.Base.create('userProfileView', Y.eZ.TemplateBasedView, [], {
+        events: {
+            '.ez-user-profile': {
+                'tap': '_toggleUserMenu'
+            }
+        },
+
+        initializer: function () {
+           this.after('*:userInfoChange', this._setUserInfo, this);
+        },
+
+        /**
+        * Renders the user profile view
+        *
+        * @method render
+        * @return {eZ.UserProfileView} the view itself
+        */
+        render: function () {
+            this.get('container')
+                .setHTML(this.template())
+                .addClass(CLASS_USER_PROFILE_HIDDEN);
+
+            this.fire('getUserInfo');
+            this._renderUserMenu();
+
+            return this;
+        },
+
+        /**
+        * Renders the user menu view
+        *
+        * @method _renderUserMenu
+        */
+        _renderUserMenu: function () {
+            var view = this.get('userMenuView');
+
+            view.addTarget(this);
+            this.get('container').append(view.render().get('container'));
+            view.set('active', true);
+        },
+
+        /**
+         * Sets name, role and image for the user
+         *
+         * @method _setUserInfo
+         * @protected
+         */
+        _setUserInfo: function () {
+            var user = this.get('userInfo'),
+                container = this.get('container'),
+                fields = user.userData.Version.Fields.field,
+                imageUrl,
+                imageAltText;
+
+                fields.forEach(function (field) {
+                    if (field.fieldDefinitionIdentifier === 'image' && field.fieldValue) {
```

Comment:
well, actually I've just realized the user response already contains the current version of the Content item of the user and after thinking a bit about it, I think the change to the User model to get the avatar should be done differently. Basically, the user model should get a new `avatar` attribute than we'll give access to the field value of the image field (if there's one). To do that, here are a needed changes:

1. add an `avatar` attribute to the user model
1. when loading the user, we should load the corresponding the content type directly, this will result in something like (not tested):
   ```js
sync: function (action, options, callback) {
    var api = options.api;

    if ( action === 'read' ) {
        api.getUserService().loadUser(this.get('id'), function (err, response) {
            var contentType;
            if ( err ) {
                return callback(err, response);
            }
            contentType = new Y.eZ.ContentType({id: response.document.User.ContentType._href});
            contentType.load({api: api}, function (error) {
                response.document._contentType = contentType;
                callback(error, response);
            });
        });

    ```
1. then in a custom `_parseStruct` method, from the Content type you can get the image field definition identifier and then get the corresponding field value in the version available in the REST User response, something like:
    ```js
        _parseStruct: function (struct, responseDoc) {
            var attrs, imageField,
                imageIdentifiers = responseDoc._contentType.getFieldDefinitionIdentifiers('ezimage');

            attrs = this.constructor.superclass._parseStruct.call(this, struct);
            imageField = struct.Version.Fields.field.filter(function (field) {
                return field.fieldDefinitionIdentifier == imageIdentifiers[0];
            });
            attrs.avatar = imageField.length ? imageField[0].fieldValue : null;
            return attrs;
        },
     ```

that way, once a user is loaded, the avatar field value is directly available in the `avatar` attribute.